undergradmath.typ

```diff
@@ -519,8 +519,8 @@ The last three here are display style.
     f: RR -> RR
     ```
   ],
-  [$ "9.8" "m/s"^2 $],
-  [`"9.8" "m/s"^2` @tricky],
+  [$ 9.8 med "m/s"^2 $],
+  [`9.8 med "m/s"^2` @tricky],
```

Comment:
Yes, the number shouldn't be quoted.

But no, the space between the number and the unit is not `med`. It is indeed `sym.space.thin` or in raw code `\u{2009}` (see #17 and also [metro]).

[metro]: https://github.com/fenjalien/metro/blob/37e353885e57279051d06f27ff6e9413843f3dd6/src/impl/qty.typ#L8

Reply:
Voici an example https://typst.app/project/r5wVO2daO1UM0dctW5eC71.